server/core/src/rodsServer.cpp

```diff
@@ -408,6 +539,8 @@ int main(int argc, char** argv)
         return 1;
     }
 
+    create_stacktrace_directory();
```

Comment:
consider 'stacktraces' ?

Reply:
i think the name of the function is clear.

i don't think the function name needs to include the directory name.

willing to be convinced though.

Reply:
just figured since it could hold more than one...

fine either way.

Reply:
leaving function name as is.

the actual directory name has been updated to be "stacktraces".